components/automate-gateway/api/iam/v2/common/policy.proto

```diff
@@ -11,11 +11,18 @@ enum Type {
 }
 
 message Policy {
+    // Name for the policy.
     string name = 1;
+    // Unique ID for the policy. Cannot be changed after creation.
     string id = 2;
+    // Whether this policy is user created or chef managed.
+    // One of `CUSTOM`, or `CHEF_MANAGED`, respectively.
```

Comment:
Should not have a comma when there are two items--this applies throughout the PR.
```suggestion
    // One of `CUSTOM` or `CHEF_MANAGED`, respectively.
```